include/ttmlir/Target/TTNN/program.fbs

```diff
@@ -163,6 +163,13 @@ table SliceOp {
   step: [int64];
 }
 
+table LinearOp {
+  in0: tt.target.TensorRef;
+  in1: tt.target.TensorRef;
+  bias: tt.target.TensorRef;  // optional
```

Comment:
By default all table fields are optional. On the other hand, we can force fields to be `required`. So I'd remove this comment to avoid confusion. Runtime can inspect if `bias` is non-null and forward it to ttnn.